x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_errors.tsx

```diff
@@ -16,12 +16,14 @@ import { i18n } from '@kbn/i18n';
 import { HttpLogic } from '../../../../../shared/http';
 
 import { ML_NOTIFICATIONS_PATH } from '../../../../routes';
+import { SendEnterpriseSearchTelemetry } from '../../../../../shared/telemetry';
 
 export const TextExpansionErrors = ({ error }: { error: { title: string; message: string } }) => {
   const { http } = useValues(HttpLogic);
 
   return (
     <>
+      <SendEnterpriseSearchTelemetry action="error" metric="textExpansionModel-error" />
```

Comment:
Not sure about the metric name, the [only references](https://github.com/search?q=repo%3Aelastic%2Fkibana+%22SendEnterpriseSearchTelemetry+action%3D%5C%22error%5C%22%22&type=code) I found use short and generic metric names.